docs/en/02_Developer_Guides/00_Model/10_Versioning.md

```diff
@@ -739,7 +739,7 @@ $record->canViewStage();
 
 #### Customising permissions for a versioned DataObject
 
-Versioned object visibility can be customised in one of the following ways by editing your user code:
+The versioned object visibility can be customised in one of the following ways by editing your user code:
```

Comment:
I don't think this should be changed. `Versioned` is the name of the feature that versions objects.